src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
```

Comment:
The `<html>` tag should not be self-closing. It should be a standard opening tag like `<html>`.